rpc/src/main/scala/org/bitcoins/rpc/jsonmodels/WalletResult.scala

```diff
@@ -172,3 +186,34 @@ case class UnspentOutput(
   spendable: Boolean,
   solvable: Boolean)
   extends WalletResult
+
+case class AddressInfoResult(
+  address: BitcoinAddress,
+  scriptPubKey: ScriptPubKey,
+  ismine: Boolean,
+  iswatchonly: Boolean,
+  isscript: Boolean,
+  iswitness: Boolean,
+  // witness_version: Option[WitnessVersion],
+  // TODO something better here
+  witness_program: Option[String],
+  // TODO Option[Script]
+  script: Option[String],
```

Comment:
Probably `Option[ScriptPubKey]`

Reply:
Possible values are:  `nonstandard, pubkey, pubkeyhash, scripthash, multisig, nulldata, witness_v0_keyhash, witness_v0_scripthash, witness_unknown`. `ScriptPubKey` is not that, if I've understood it correctly?